v1.0-rc.2.md

```diff
@@ -50,13 +50,16 @@ Get future release notes emailed to you:
 - The number of transactions that would be affected by a change in isolation level is now tracked. [#15234](https://github.com/cockroachdb/cockroach/pull/15234)
 - Improved style of the single node and log pages. [#15486](https://github.com/cockroachdb/cockroach/pull/15486)
 - The event log now refreshes automatically. [#15502](https://github.com/cockroachdb/cockroach/pull/15502)
+- Add tooltips for graphs on overview page. [#15624](https://github.com/cockroachdb/cockroach/pull/15624)
 
 ### Bug Fixes
 
 - Fixed a panic when a query returning a constant `NULL` used the new distributed execution engine. [#15530](https://github.com/cockroachdb/cockroach/pull/15530)
 - Fixed two issues that could cause requests to get stuck indefinitely. [#15557](https://github.com/cockroachdb/cockroach/pull/15557) [#15573](https://github.com/cockroachdb/cockroach/pull/15573)
 - Fixed a rare crash when extending table leases. [#15560](https://github.com/cockroachdb/cockroach/pull/15560)
 - Fixed a crash when a transaction that uses `SAVEPOINT cockroach_restart` does something other than `ROLLBACK TO SAVEPOINT` after an error. [#15563](https://github.com/cockroachdb/cockroach/pull/15563)
+- Fixed `RESTORE` for interleaved tables. [#15564](https://github.com/cockroachdb/cockroach/pull/15564)
+- Return parse error instead of panic when constant typechecking fails. [#15636](https://github.com/cockroachdb/cockroach/pull/15636)
```

Comment:
well if the table is empty (or, if the column is part of the original CREATE TABLE), that was fine before too, so I said something about backfilling to not scare people too much.